src/thebelab.js

```diff
@@ -351,9 +351,9 @@ export function requestBinder({
     // trailing / on binderUrl
     binderUrl = binderUrl.replace(/(\/?$)/g, "");
     //add /.git if not present
-    if (!repo.endsWith("/.git")) {
```

Comment:
I'm not sure when this got added, but I don't think we should be adding to the URL. Let's leave it exactly as the user specified.

Reply:
as in, you think we should remove this whole `if` statement?

Reply:
exactly